object/host_system.go

```diff
@@ -32,12 +32,31 @@ type HostSystem struct {
 	InventoryPath string
 }
 
+func (h HostSystem) String() string {
```

Comment:
My latest commit implements `String()` as you suggested, with the addition of also printing the `InventoryPath` if it is set. This does not require network traffic and adds useful data.